installer/templates/phx_web/router.ex

```diff
@@ -31,18 +40,10 @@ defmodule <%= web_namespace %>.Router do
   if Mix.env() == :dev do
```

Comment:
Yeah, given the warning, I would go with:

```suggestion
  if Mix.env() in [:dev, :test] do
```